docs/en/siem/siem-ui.asciidoc

```diff
@@ -2,13 +2,174 @@
 [role="xpack"]
 == SIEM UI
 
+IMPORTANT: This document is a work in progress. 
+It covers features scheduled for beta release in 7.2.
+
 beta[]
 
-After you have security monitoring <<install-siem,up and
-running>> and data is streaming to {es}, use the {siem-ui} in {kib} to monitor
-and identify security problems in real time.
+After you have security monitoring <<install-siem,up and running>> and data is
+streaming to {es}, check out the {siem-ui} in {kib}. You can monitor, identify,
+track, and notate potential security issues in real time, or look back
+historically to build a case.
+
+The {siem-app} is powerful, flexible, and highly interactive. The UI elements
+are designed to be discoverable, clickable, draggable and dropable, expandable
+and collapsible, resizable, moveable, and so forth.
+
+The *{kib} Query Language (KQL) bar* is available throughout the {siem-app} for
+searching and filtering.
+
+[float]
+[[siem-overview-ui]]
+=== Overview
+
+The Overview page provides a high-level view into your security status. It can
+help surface activity that requires further investigation.
+
+Notice the *Timeline* on the right of the {siem-ui}. It is always available in its
+collapsed or expanded state. Identify items you want to dig into, and drag and
+drop them there. Those items are waiting for you when you are ready to start your
```

Comment:
There's been a lot of discussion around the naming of the `{timeline}` feature in the last year. If there's ever been a thing you wanted to protect from an eventual rename with a curly tag, I think it's `{timeline}` :-)